app/serializers/current_room_serializer.rb

```diff
@@ -24,4 +25,8 @@ def owner_name
   def owner_avatar
     user_avatar(object.user)
   end
+
+  def last_session
+    object.last_session.strftime('%A %B %e, %Y %l:%M%P')
```

Comment:
I think this is fine for now, but we need to refer back to this in the future.
Since, that will make it hard for us to localize the datetime and time zone it,
I think a better approach is to serve a timestamp and leave the formatting responsibility to the frontend app.